realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -260,6 +260,8 @@ private static void checkFilesDirAvailable(Context context) {
      * @throws RealmMigrationNeededException if no migration has been provided by the default configuration and the
      * RealmObject classes or version has has changed so a migration is required.
      * @throws RealmFileException if an error happened when accessing the underlying Realm file.
+     * @throws io.realm.exceptions.DownloadingRealmInterruptedException if {@link SyncConfiguration.Builder#waitForInitialRemoteData()}
```

Comment:
Is full package needed? `RealmMigrationNeededException` and `RealmFileException` doesn't start with `io.realm.exceptions`.